src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -83,9 +95,67 @@
     }
 @end
 
+@private clientStreamingMethodSampleCode(apiMethod)
+    $requests = [$request];
+    
+    // Write data to server and wait for a response
+    $streamingResponse = {@methodCallSampleCode(apiMethod)};
+    $result = $streamingResponse->writeAllAndWait($requests);
+    // doSomethingWith($result)
+    
+    // OR write data as it becomes available, then wait for a response
+    $streamingResponse = {@methodCallSampleCode(apiMethod)};
+    foreach (read_requests_from_stream() as $request) {
```

Comment:
yeah, the `$requests` array would be much clearer.

I think we should test what we have written, so if you are adding support for client streaming, add a baseline.